modules/react/package.json

```diff
@@ -40,7 +40,8 @@
     "prop-types": "^15.8.1",
     "styled-components": "^4.4.1",
     "styled-react-modal": "^3.1.1",
-    "supercluster": "^8.0.1"
+    "supercluster": "^8.0.1",
+    "boxicons": "^2.1.4"
```

Comment:
Why are we adding `boxicons`? Was it missing? Kind of unrelated to this PR...

Reply:
Because, the modules/react/src/components/icon.tsx use `boxicons `. If we don't install this library, we have an error (You can see the description) when running test